.github/workflows/create-github-release.yaml

```diff
@@ -47,6 +47,15 @@ jobs:
       version_tag: ${{ github.ref_name }}
     secrets: inherit
 
+  build-windows-artifacts:
```

Comment:
I think you also need to update `check_release_assets_uploaded.py` to make sure the release waits until these are completed

Reply:
Oh that's a sneaky script that I missed. Thanks for pointing that out.

Added.